### PR TITLE
Corrected the apps directory.

### PR DESCRIPTION
**Pull request by EParzefall**
Fixed the apps directory name and added the docker directory as reported in issue #3393. Backport for 10.7 and 10.6 needed.

### CODE REVIEW

File: modules/developer_manual/pages/core/theming.adoc [hunk 1]
```diff
@@ -17,15 +17,14 @@ NOTE: Before ownCloud 10, theming was done via the `config.php` entry `'theme' =
...
-(adjust the names and paths accordingly)
+The following 7 commands get a theme installed quickly. Adjust the names and paths according to your setup. For docker installations use the directory `/var/www/owncloud/custom`.
 
 ----
 # 0 You need unzip
 apt install unzip
 
-# 1. Go in your apps folder
-cd /var/www/owncloud/apps
+# 1. Go into your apps folder
```
Comment (mmattel) [inline modules/developer_manual/pages/core/theming.adoc:26]:
```suggestion
# 1. If exists, go into your `apps-external` folder, else use `apps`
```
`apps-external` ist im normalfall vorhanden, muß aber nicht

File: modules/developer_manual/pages/core/theming.adoc [hunk 2]
```diff
@@ -17,15 +17,14 @@ NOTE: Before ownCloud 10, theming was done via the `config.php` entry `'theme' =
 This is deprecated in ownCloud 10.
 Users who have this entry in their `config.php` should remove it and use a theme app to customize their ownCloud instance instead.
 
-Here are 7 commands to get a theme quickly:
-(adjust the names and paths accordingly)
+The following 7 commands get a theme installed quickly. Adjust the names and paths according to your setup. For docker installations use the directory `/var/www/owncloud/custom`.
 
 ----
 # 0 You need unzip
 apt install unzip
```
Comment (mmattel) [inline modules/developer_manual/pages/core/theming.adoc:24]:
```suggestion
sudo apt install unzip
```
apt install ist ein priveledged command

File: modules/developer_manual/pages/core/theming.adoc [hunk 3]
```diff
@@ -17,15 +17,14 @@ NOTE: Before ownCloud 10, theming was done via the `config.php` entry `'theme' =
 This is deprecated in ownCloud 10.
 Users who have this entry in their `config.php` should remove it and use a theme app to customize their ownCloud instance instead.
 
-Here are 7 commands to get a theme quickly:
-(adjust the names and paths accordingly)
+The following 7 commands get a theme installed quickly. Adjust the names and paths according to your setup. For docker installations use the directory `/var/www/owncloud/custom`.
 
 ----
```
Comment (mmattel) [inline modules/developer_manual/pages/core/theming.adoc:22]:
```suggestion
[source,console,subs="attributes+"]
----
```

**Review by mmattel — APPROVED** [review]

File: modules/developer_manual/pages/core/theming.adoc [hunk 4]
```diff
@@ -17,15 +17,17 @@ NOTE: Before ownCloud 10, theming was done via the `config.php` entry `'theme' =
 This is deprecated in ownCloud 10.
 Users who have this entry in their `config.php` should remove it and use a theme app to customize their ownCloud instance instead.
 
-Here are 7 commands to get a theme quickly:
-(adjust the names and paths accordingly)
+== Quick Start
 
+For the impatent, who want to play with themes right away, the following 8 commands get a theme installed quickly. Adjust the names and paths according to your setup. For docker installations use the directory `/var/www/owncloud/custom`.
```
Comment (phil-davis) [inline modules/developer_manual/pages/core/theming.adoc:22]:
```suggestion
For the impatient, who want to play with themes right away, the following 8 commands get a theme installed quickly. Adjust the names and paths according to your setup. For docker installations use the directory `/var/www/owncloud/custom`.
```

**Review by phil-davis — APPROVED** [review]
I squashed the commits.

**Comment by phil-davis** [comment]
Backports:
10.7 #3403 
10.6 #3404

**Pull request merged**